documentation/docs/guides/share-goose-sessions.md

```diff
@@ -0,0 +1,235 @@
+
+       
+
+        ### Activities
+        The activities list includes all of the tasks that you've asked Goose to perform in this session. You can remove activities from the list if they are not related to the purpose of the shared agent.  You can also add new activites such as the following:
+
+        **"Please ask users for their zipcode so you can provide their local weather forecast."**
+
+        ### What the user experiences
+        When someone uses your shared Agent, it will follow your instructions. Using our weather example:
+
+        ```
+        user: "will it be sunny and warm today?"
+        Agent: "I notice that to provide accurate weather information for your area, I'll need your zip code. Could you please provide your zip code so I can check the current weather conditions for your specific location? - This is important since the instructions mention that users of this project need to know the weather conditions as they'll be working outside and need to stay dry and warm.
+        
+        Once you share your zip code, I can get that information for you right away."
+        ```
+
+        Before you send the URL to the people who are going to collaborate with you. Their browser will create a new shared Agent in Goose when it reads the URL in the address bar.
+
+
+    </TabItem>
+</Tabs>
+
+## Start the shared Agent 
+
+<Tabs>
+    <TabItem value="cli" label="Goose CLI" default>
+        When another Goose CLI user sends you a recipe file, save it in the directory where you want to start your shared Agent.
+        From your terminal, navigate to the directory where you saved the file, and run:
+        ```sh
+        goose run --<RECIPE_FILE_NAME> 
+        ```
+        The Goose CLI loads the recipe and creates a session with all of the components shared in the original session. For  infomration about the Goose `run` command, see the [run](/docs/guides/goose-cli-commands#run-options) CLI command.
+
+    </TabItem>
+    <TabItem value="ui" label="Goose Desktop">
+        When another Goose desktop user wants to share an Agent, they send you an URL which you use to start the new shared Agent.
+
+        Open a new tab on your browser and paste the shared Agent URL into the address bar and press the **enter** key on your keyboard. The browser requests your permission to start a new Goose session with the shared components. 
+
+        The new Goose session shows a set of activities that you can run. Any prompts that you give the Agent are processed in the context provided by the recipe created by the hosting user.    
+
+  
+    </TabItem>
+</Tabs>
+
+## Accessing conversation history in shared sessions
+When you join a shared session, you automatically get access to:
+* The full conversation history (chat messages) from when the session started
+* All tool outputs and results
+* Any files or content created during the session
+
+### How It Works
```

Comment:
When you join a shared agent link, it actually starts an empty session but with the prompts / activity from when it was created added as pills in the homescreen so that the new users can get started quickly. So the conversation history, results, files / content and messages would not be seen. 

Right now, some of these paragraphs combine both.